README.md

```diff
@@ -26,9 +26,16 @@ Notification must be sent when a new report is available.
 
 List the dependencies of the Analysis-functionality.
 
-1. Access to the Server containing the telemetrics in a csv file
-1. _enter dependency
-1. _enter dependency
+1. Credentials info and Access to the Server containing the telemetrics in a csv file.
+2. Read Access to the Battery telemetrics data in a csv file.
```

Comment:
yes in precise it can be considered as CSV file reader dependency, i have mentioned all kind of dependencies for csv file access from server and reading